toolbox/util/commonUtils.go

```diff
@@ -86,9 +87,29 @@ func ContainsString(slice []string, target string) bool {
 	return false
 }
 
-// UpdateKeyValueInFile updates in the file all occurrences of key to
+// updateKeyValueInTomlLines updates all occurrences of key to a new value
+func updateKeyValueInTomlLines(lines []string, key, value string) ([]string, bool) {
+	// toml dependecies are of the form
+	//  name = "istio.io/api"
+	//  revision = "b08011c721e03edd61c721e4943607c97b7a9879"
+
+	found := false
+	keySearch := fmt.Sprintf("name = %q", key)
+	for i, line := range lines {
+		if strings.HasPrefix(line, "#") || strings.HasPrefix(line, "//") {
+			continue
+		}
+		if strings.Contains(line, keySearch) {
```

Comment:
Is it worthwhile to also verify that lines[i+1] has a prefix of "  revision ="?  It looks like other attributes could be specified like "version" or "source".

Reply:
fixed, by crashing if revision not found